shoop/utils/pdf_export.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# This file is part of Shoop.
+#
+# Copyright (c) 2012-2016, Shoop Ltd. All rights reserved.
+#
+# This source code is licensed under the AGPLv3 license found in the
+# LICENSE file in the root directory of this source tree.
+from django.http import HttpResponse
+
+try:
+    import weasyprint
+except ImportError:
+    weasyprint = None
+
+
+def render_html_to_pdf(html):
+    if not weasyprint:
+        raise NotImplementedError("weasyprint is not available")
```

Comment:
This isn't an implementation error. Would be better to raise a proper Problem that informs the user that the required library should be installed.